app/src/main/java/test1/daihocbachkhoa/tranphuc1995/assignment1/Beverages.java

```diff
@@ -0,0 +1,30 @@
+package test1.daihocbachkhoa.tranphuc1995.assignment1;
+
+/**
+ * Created by tran_phuc on 5/16/2016.
+ */
+public class Beverages {
+    private String drinkName;
+    private int imgBeverages;
```

Comment:
`imgBeverages` Did you mean this is `image-resource-id`? If yes, please change it to a more meaningful name.